src/Microsoft.ApplicationInsights.AspNetCore/ExceptionTrackingMiddleware.cs

```diff
@@ -1,42 +0,0 @@
-﻿namespace Microsoft.ApplicationInsights.AspNetCore
-{
-    using System;
-    using System.Threading.Tasks;
-    using Microsoft.ApplicationInsights.DataContracts;
-    using Microsoft.ApplicationInsights.Extensibility.Implementation;
-    using Microsoft.AspNetCore.Http;
-
-    /// <summary>
-    /// Sends telemetry about exceptions thrown by the application to the Microsoft Application Insights service.
-    /// </summary>
-    public sealed class ExceptionTrackingMiddleware
```

Comment:
As a minimum let's keep it as "Obsolete". However I'd recommend to keep it as is in case customer has own error processing middleware and neither unhandled exception nor diagnostics middleware will catch it